backend/src/airtable-api.js

```diff
@@ -168,3 +169,42 @@ export const fetchSunriseBills = async () => {
   }
   return { bills, campaigns };
 };
+
+export const fetchSingleBill = (printNumber) => {
+  return new Promise((resolve, reject) => {
+    base('ImportedScorecard')
+      .select({
+        view: 'Grid view',
+        fields: ["Campaign", "Importance"],
+        maxRecords: 1,
+        filterByFormula: `{Senate Number} = '${printNumber}'`,
+      })
+      .firstPage((err, records) => {
+        if (err) {
+          console.error('Airtable fetch error:', err);
+          return reject(err); // Reject the Promise if there's an error
+        }
+        
+        if (records && records.length > 0) {
+          const record = records[0];
+          const bill = {
+            campaign: record.get("Campaign"),
+            importance: record.get("Importance"),
+          };
+          resolve(bill); // Resolve with the bill data
+        } else {
+          resolve(null); // Resolve with null if no record is found
+        }
+      });
+  });
+};
+
+export const fetchSingleCampaign = (campaignId) => {
+  return  new Promise((resolve, reject) => {
+    base('Campaigns').find(`${campaignId}`, function(err, campaign) {
+      if (err) { console.error(err); return reject(err); }
+      // console.log(campaign);
+      resolve(campaign); // Resolve with the campaign data
+    });
+  });
+};
```

Comment:
small nit, but could you make sure the changes are linted?

Reply:
I am having issues running ESLint for Bill.jsx, airtable-api.js, and app.js (could not find config file)...

Reply:
has been checked now. does it look good to you?